tests/_support/AcceptanceTester.php

```diff
@@ -487,4 +487,12 @@ public function checkTemplateIsPresent(int $templateIndex, string $templateCateg
     $i->waitForElement($template);
     return $template;
   }
+
+  public function clearFormField(string $selector) {
```

Comment:
With this implementation, it will work only for inputs that have up to 3 characters. I think you could use https://codeception.com/docs/modules/WebDriver#grabattributefrom method and based on value length press backspace as many times as needed.